packages/react/test/specs/components/Accordion/Accordion-test.tsx

```diff
@@ -0,0 +1,227 @@
+      const accordion = wrapper.find(Accordion)
+      getTitleButtonAtIndex(wrapper, 1).simulate('click')
+      expect(accordion.state('focusedIndex')).toEqual(1)
+    })
+
+    it('is changed by arrow key navigation', () => {
+      const wrapper = mountWithProvider(<Accordion panels={panels} />)
+      const accordion = wrapper.find(Accordion)
+      getTitleButtonAtIndex(wrapper, 1).simulate('click')
+      getTitleButtonAtIndex(wrapper, 1).simulate('keydown', {
+        keyCode: keyboardKey.ArrowUp,
+        key: 'ArrowUp',
+      })
+      expect(accordion.state('focusedIndex')).toEqual(0)
+
+      getTitleButtonAtIndex(wrapper, 0).simulate('keydown', {
+        keyCode: keyboardKey.ArrowDown,
+        key: 'ArrowDown',
+      })
+      expect(accordion.state('focusedIndex')).toEqual(1)
+    })
+
+    it('is changed by arrow key navigation in a circular way', () => {
+      const wrapper = mountWithProvider(<Accordion panels={panels} />)
+      const accordion = wrapper.find(Accordion)
+      getTitleButtonAtIndex(wrapper, 0).simulate('click')
+      getTitleButtonAtIndex(wrapper, 0).simulate('keydown', {
+        keyCode: keyboardKey.ArrowUp,
+        key: 'ArrowUp',
+      })
+      expect(accordion.state('focusedIndex')).toEqual(panels.length - 1)
+
+      getTitleButtonAtIndex(wrapper, panels.length - 1).simulate('keydown', {
+        keyCode: keyboardKey.ArrowDown,
+        key: 'ArrowDown',
+      })
+      expect(accordion.state('focusedIndex')).toEqual(0)
+    })
+
+    it('is changed to `0` at Home keydown', () => {
+      const wrapper = mountWithProvider(<Accordion panels={panels} />)
+      const accordion = wrapper.find(Accordion)
+      getTitleButtonAtIndex(wrapper, 2).simulate('click')
+      getTitleButtonAtIndex(wrapper, 2).simulate('keydown', {
+        keyCode: keyboardKey.Home,
+        key: 'Home',
+      })
+      expect(accordion.state('focusedIndex')).toEqual(0)
+    })
+
+    it('is changed to last index at End keydown', () => {
+      const wrapper = mountWithProvider(<Accordion panels={panels} />)
+      const accordion = wrapper.find(Accordion)
+      getTitleButtonAtIndex(wrapper, 0).simulate('click')
+      getTitleButtonAtIndex(wrapper, 0).simulate('keydown', {
+        keyCode: keyboardKey.End,
+        key: 'End',
+      })
+      expect(accordion.state('focusedIndex')).toEqual(panels.length - 1)
+    })
+
+    it('focuses the button element when is changed via focus handler', () => {
+      const wrapper = mountWithProvider(<Accordion panels={panels} />)
+      const title = getTitleButtonAtIndex(wrapper, 1)
+      title.simulate('click')
+      title.simulate('keydown', { keyCode: keyboardKey.ArrowUp, key: 'ArrowUp' })
+      expect(document.activeElement).toEqual(getTitleButtonAtIndex(wrapper, 0).getDOMNode())
+    })
+  })
+
+  describe('panels', () => {
+    it('when clicked call onClick and onTitleClick if provided by the user', () => {
```

Comment:
onClick should be tested by the isConformant - handles events transparently. The test for the onTitleClick is okay (although not sure why we have dedicated prop for that, but as it was not added by this changes, would open the discussion separately).

Reply:
same here